### PR TITLE
Partial error upload offline conversion

### PR DESCRIPTION
**Pull request by PierrickVoulet**

### CODE REVIEW

**Comment by PierrickVoulet** [comment]
Partial errors are currently not supported for conversions, I opened #250 to fix the issue as a separated PR.

File: examples/Remarketing/UploadOfflineConversion.php [hunk 1]
```diff
@@ -140,22 +176,69 @@ public static function runExample(
...
-            $uploadedClickConversion->getConversionActionUnwrapped(),
-            PHP_EOL
-        );
+    /**
+     * Check if there exists partial failure error in the given upload click conversions response.
+     *
+     * @param UploadClickConversionsResponse $response the upload click conversions response
+     */
+    private static function checkIfPartialFailureErrorExists(
+        UploadClickConversionsResponse $response
```
[the remark this answers is not in the document]
Comment (PierrickVoulet) [reply examples/Remarketing/UploadOfflineConversion.php]: It was already imported before this refactoring (see [here](https://github.com/googleads/google-ads-php/pull/249/files/17b87c9cbd2e61564a48f3ecf88444ed09bfaf35#diff-19cbd75cd61f5ca8369cbef69fccb57eR35)), do you mean something else?

File: examples/Remarketing/UploadOfflineConversion.php [hunk 2]
```diff
@@ -140,22 +176,69 @@ public static function runExample(
...
+    ) {
+        if (!is_null($response->getPartialFailureError())) {
+            printf("Partial failures occurred. Details will be shown below.%s", PHP_EOL);
+        } else {
+            printf(
+                "All operations completed successfully. No partial failures to show.%s",
+                PHP_EOL
+            );
```
[the remark this answers is not in the document]
Comment (PierrickVoulet) [reply examples/Remarketing/UploadOfflineConversion.php]: I get 103 characters if I try.

File: examples/Remarketing/UploadOfflineConversion.php (hunk 1 above)
[the remark this answers is not in the document]
Comment (fiboknacky) [reply examples/Remarketing/UploadOfflineConversion.php]: Sorry, I missed that.

**Pull request merged**